tests/run/returns.scala

```diff
@@ -0,0 +1,16 @@
+def foo(x: Int): Int =
+  if x == 1 then
+    return
+      x
+  else
+    2
+
+def bar(): Unit =
```

Comment:
Is this equivalent to:
```scala
def bar(): Unit = 
    return assert(false)
```
?

Reply:
No; assert(false) is dead code in that case.